paasta_tools/utils.py

```diff
@@ -800,6 +800,13 @@ def get_sensu_port(self):
         """
         return int(self.get('sensu_port', 3030))
 
+    def get_dockerfile_location(self):
```

Comment:
This is identical to the function in `marathon_tools.py`. We should only have this in one file and then import it as necessary everywhere else.